toolchain/sem_ir/node_kind.def

```diff
@@ -40,8 +40,6 @@
 #error "Must define the x-macro to use this file."
 #endif
 
-CARBON_SEMANTICS_NODE_KIND_IMPL(Invalid, "invalid", None, NotTerminator)
```

Comment:
Reminder: consider moving the field information from the NodeData namespace node.h to this file in a future PR. This would keep all the information together and remove redundancy.